spark/src/main/scala/ai/chronon/spark/SparkSessionBuilder.scala

```diff
@@ -117,7 +119,7 @@ object SparkSessionBuilder {
     }
     val spark = builder.getOrCreate()
     // disable log spam
-    spark.sparkContext.setLogLevel("ERROR")
+    // spark.sparkContext.setLogLevel("ERROR")
```

Comment:
do you want to keep this line?

Reply:
There is something wrong with our logs - they just disappear in the tests